packages/cli/test/fixtures/unit/vercel-env-pull/.gitignore

```diff
@@ -0,0 +1,3 @@
+.next
+yarn.lock
+.vercel
```

Comment:
Maybe this should be more specific since you intended to include `.vercel/project.json`?

Reply:
I removed the line. The updated PR matches the intent better.